README.md

```diff
@@ -25,6 +25,30 @@ Simply install caustics from PyPI:
 pip install caustics
 ```
 
+## Minimal Example
+
+```python
+import matplotlib.pyplot as plt
+import caustics
+
+C = caustics.cosmology.FlatLambdaCDM()
+
+sie = caustics.lenses.SIE(
+    cosmology=C, z_l=0.5, x0=0.0, y0=0.0, q=0.4, phi=1.5708, b=1.0
+)
+src = caustics.light.Sersic(x0=-0.2, y0=0.0, q=0.6, phi=-0.785, n=1.5, Re=3.0, Ie=1.0)
+
+sim = caustics.sims.Lens_Source(
+    lens=sie, source=src, pixelscale=0.05, pixels_x=100, z_s=1.5
+)
+
+plt.imshow(sim(quad_level=3).detach().cpu().numpy(), origin="lower")
```

Comment:
I don't know that we should put quad_level in the basic example. I think it would be more clear without.

Reply:
I would agree, except if we are assuming people will copy/paste this example to work with, we would want them to get accurate fluxes. The difference between direct sampling and `quad_level=3` is actually quite significant... Someone copy/pasting may not think about their integration depth.